### PR TITLE
Added Umar khatab's Profile

### PR DESCRIPTION
**Pull request by umaresso**
## Changes proposed
Added my name in the list

### CODE REVIEW

File: public/data/umaresso.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Umar Khatab",
+  "type": "personal",
+  "bio": "React Js developer",
```
Comment (KendallDoesCoding) [inline public/data/umaresso.json:4]:
```suggestion
  "bio": "React JS developer",
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by schmelto — CHANGES_REQUESTED** [review]
Please do not add `package-lock.json` to your pr if you need any help please let us know

File: public/data/umaresso.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/umaresso.json:4]: @umaresso You can also replace it with Javascript, but for now its ok as it is, just capitalize the text.

**Comment by KendallDoesCoding** [comment]
> Please do not add `package-lock.json` to your pr if you need any help please let us know

As per commit history, he didn't add this, He made this PR in the main branch though so maybe that's the issue?

File: public/data/umaresso.json (hunk 1 above)
Comment (umaresso) [reply public/data/umaresso.json]: okey , i have done the suggested changes. Now i'm going to close the issue and start a new One

**Pull request closed without merging**